counsel.el

```diff
@@ -3207,11 +3212,6 @@ A is the left hand side, B the right hand side."
   :group 'ivy
   :type 'string)
 
-(defcustom counsel-yank-pop-height 5
```

Comment:
Probably best to mark this as obsolete and point to `ivy-height-alist` instead via `define-obsolete-variable-alias`.